desktop/faqs.md

```diff
@@ -0,0 +1,215 @@
+
+### How do I ensure that all users on my team are using the same version?
+
+Previously you had to manage this yourself: now it will happen automatically as a side effect of all users being on the latest version.
+
+### My colleague has got a new version but I haven’t got it yet.
+
+Sometimes we may roll out a new version gradually over a few days. Therefore, if you wait, it will turn up soon. Alternatively, you can select **Check for Updates** from the Docker menu to jump the queue and get the latest version immediately.
+
+### Where can I find information about Stable and Edge releases?
+
+Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single, cumulative release stream for all users.
+
+## Support
+
+### Does Docker Desktop offer support?
+
+Yes, Docker Desktop offers support for Pro and Team users. For more information, see [Docker Desktop Support](../docker-for-mac/troubleshoot.md#support).
+
+For information about the pricing plans and to upgrade your existing account, see [Docker pricing](https://www.docker.com/pricing){: target="_blank" rel="noopener" class="_"}.
+
+### What kind of feedback are you looking for?
+
+Everything is fair game. We'd like your impressions on the download-install
+process, startup, functionality available, the GUI, usefulness of the app,
+command line integration, and so on. Tell us about the issues you are experiencing, what you like, or request a new feature through our public [Docker Roadmap](https://github.com/docker/roadmap){: target="_blank" rel="noopener" class="_"}.
+
+### How is personal data handled in Docker Desktop?
+
+When uploading diagnostics to help Docker with investigating issues, the uploaded diagnostics bundle may contain personal data such as usernames and IP addresses. The diagnostics bundles are only accessible to Docker, Inc.
+employees who are directly involved in diagnosing Docker Desktop issues.
+
+By default Docker, Inc. will delete uploaded diagnostics bundles after 30 days. You may also request the removal of a diagnostics bundle by either specifying the diagnostics ID or via your GitHub ID (if the diagnostics ID is mentioned in a GitHub issue). Docker, Inc. will only use the data in the diagnostics bundle to investigate specific user issues, but may derive high-level (non personal) metrics such as the rate of issues from it.
+
+## Mac FAQs
+
+### What is Docker.app?
+
+`Docker.app` is Docker Desktop on Mac. It bundles the Docker client and Docker Engine. `Docker.app` uses the macOS Hypervisor.framework to run containers.
+
+### Is Docker Desktop compatible with Apple silicon processors?
+
+At the moment, Docker Desktop is compatible with Intel processors only. You can follow the status of Apple Silicon support in our [roadmap](https://github.com/docker/roadmap/issues/142){:target="_blank" rel="noopener" class="_"}.
```

Comment:
@stephen-turner should we say something about «it’s coming soon»?

Reply:
Also, need to add a link to the docs page.

Reply:
Added a link to Apple M1 Tech Preview page.